Documentation/ProjectFile/prj/processes/process/RICHARDS_FLOW/material_property/porous_medium/porous_medium/a_id.md

```diff
@@ -0,0 +1,2 @@
+It specifies the material ID of a porous medium. The material IDs of elements
+should be given in the mesh's cell data array "MaterialIDs".
```

Comment:
Yes, without data array 'MaterialIDs'  would not lead to crash, it will return 0 for all material_id instead. The document will be updated.